src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -29,10 +33,12 @@ const defaultProps = {
 };
 
 function ReportActionItemMessage(props) {
+    const messages = _.compact(props.action.previousMessage || props.action.message);
+    const isAttachment = ReportUtils.isReportMessageAttachment(_.last(messages));
     return (
-        <View style={[styles.chatItemMessage, ...props.style]}>
+        <View style={[styles.chatItemMessage, !props.displayAsGroup && isAttachment ? styles.mt2 : {}, ...props.style]}>
```

Comment:
It's a minor thing probably, but why was `undefined` changed to `{}`? Was there any observable difference in behavior, or is it a stylistic thing?